### DISCUSSION TITLE
Installing dependencies

### DISCUSSION THREAD

**Discussion by root-dm** [codemagic-ci-cd/codemagic-docs#1193] [Q&A] [answered]
== Install Flutter dependencies ==

Build failed :|
Failed to install dependencies for pubspec file in /Users/builder/clone. Directory was not found

63848d2b543f9fb40ee4ccfc

**Comment by root-dm** [comment, marked as answer]
Fixed! Refreshed project path and selected the correct

**Reply by himesh-cm** [reply]
👍

**Comment by gordon-git** [comment]
Seems like I got the same problem.
`Failed to install dependencies for pubspec file in /Users/builder/clone. Directory was not found`
I'm new to codemagic, how did you refresh the project path?

**Reply by himesh-cm** [reply]
Hi @gordon-git ,
Please open a new discussion along with your build id, so i can look into the issue further.
